exporters/variables-scss/src/generators/contentGenerator.ts

```diff
@@ -30,15 +35,17 @@ export const generateFileContent = (
       const filteredTokens = filterTokensByTypeAndGroup(tokens, tokenType, group);
 
       // Generate css tokens
-      cssTokens += generateCssFromTokens(
-        filteredTokens,
-        mappedTokens,
-        tokenGroups,
-        group,
-        hasParentPrefix,
-        sortByNumValue,
-      );
-      cssTokens += '\n\n';
+      if (tokenType !== TokenType.typography) {
```

Comment:
This condition is also repeated in the previous function. What about making a function from this and using it in both places?

Reply:
As discussed in office, we will leave it like this in this file